tools/advisor/advisor/ini_parser.py

```diff
@@ -0,0 +1,67 @@
+from enum import Enum
```

Comment:
Add facebook headers to all files. e.g., https://github.com/facebook/rocksdb/blob/master/utilities/transactions/write_prepared_txn.h